src/prefs.ts

```diff
@@ -117,114 +133,123 @@ function settings_dialog_view(): [AppWidgets, Gtk.Container] {
         margin_end: 10,
         margin_bottom: 10,
         margin_top: 10,
-    })
+    });
 
     const win_label = new Gtk.Label({
         label: "Show Window Titles",
         xalign: 0.0,
-        hexpand: true
-    })
+        hexpand: true,
+    });
 
     const snap_label = new Gtk.Label({
         label: "Snap to Grid (Floating Mode)",
-        xalign: 0.0
-    })
+        xalign: 0.0,
+    });
 
     const smart_label = new Gtk.Label({
         label: "Smart Gaps",
-        xalign: 0.0
-    })
+        xalign: 0.0,
+    });
+
+    const unstack_label = new Gtk.Label({
+        label: "Automatically destroy single-window stacks",
```

Comment:
The label here still needs to be updated to match what https://github.com/pop-os/shell/pull/1657/commits/6adc8b28654737346aa1bec672883ee4c390b833 changed.

Reply:
Did not notice this, thanks!
![image](https://github.com/pop-os/shell/assets/58280776/42b08260-ec49-4b25-8166-f26de93b63d9)